### PR TITLE
Remove column from table

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: lambda/src/main/resources/db/migration/V13__drop-metadata-created-date.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE "ClientFileMetadata" DROP COLUMN "CreatedDate"
```
Comment (TomJKing) [inline lambda/src/main/resources/db/migration/V13__drop-metadata-created-date.sql:1]: Missing line break

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.33-SNAPSHOT"
+version in ThisBuild := "0.0.34-SNAPSHOT"
```
Comment (TomJKing) [inline version.sbt:1]: Isn't this handled by the Jenkins build now. It creates a new PR for the version bump?

**Review by TomJKing — COMMENTED** [review]
Couple of minor comments

File: version.sbt (hunk 2 above)
Comment (MancunianSam) [reply version.sbt:1]: It is but I needed it deployed so the tests wouldn't fail on the other builds so I deployed it manually :grimacing:

File: lambda/src/main/resources/db/migration/V13__drop-metadata-created-date.sql (hunk 1 above)
Comment (MancunianSam) [reply lambda/src/main/resources/db/migration/V13__drop-metadata-created-date.sql]: I've added it in.

File: version.sbt (hunk 2 above)
Comment (TomJKing) [reply version.sbt:1]: Tut, tut

**Review by TomJKing — APPROVED** [review]
Couple of minor comments

**Pull request merged**